buddypress/groups/single/edit.php

```diff
@@ -168,72 +165,72 @@
                     </div>
 					<div class="create-group__input-container create-group__input-container--vertical-spacing create-group__input-container--50">
-						<label class="create-group__label"  for="group-matrix"><?php print __("Matrix", "community-portal"); ?></label>
+						<label class="create-group__label"  for="group-matrix"><?php _e('Matrix', 'community-portal'); ?></label>
 						<input type="text" placeholder="room-alias:domain" name="group_matrix" id="group-matrix" class="create-group__input create-group__input--inline"  value="<?php print isset($form['group_matrix']) ? $form['group_matrix'] : ''; ?>"/>
 						<div class="form__error-container form__error-container--checkbox">
-							<div class="form__error"><?php print __("Please format as room-alias:domain", "community-portal"); ?></div>
+							<div class="form__error"><?php _e('Please format as room-alias:domain', 'community-portal'); ?></div>
 						</div>
                     </div>
                 </div>
                 <div class="create-group__input-row">
                     <div class="create-group__input-container create-group__input-container--vertical-spacing create-group__input-container--50">
-						<label class="create-group__label" for="group-facebook"><?php print __("Facebook", "community-portal"); ?></label>
+						<label class="create-group__label" for="group-facebook"><?php _e('Facebook', 'community-portal'); ?></label>
 						<input placeholder="https://" type="text" name="group_facebook" id="group-facebook" class="create-group__input create-group__input--inline"  value="<?php print isset($form['group_facebook']) ? $form['group_facebook'] : ''; ?>"/>
                     </div>
                     <div class="create-group__input-container create-group__input-container--vertical-spacing create-group__input-container--50">
-						<label class="create-group__label" for="group-twitter"><?php print __("Twitter", "community-portal"); ?></label>
+						<label class="create-group__label" for="group-twitter"><?php _e('Twitter', 'community-portal'); ?></label>
 						<input placeholder="https://" type="text" name="group_twitter" id="group-twitter" class="create-group__input create-group__input--inline"  value="<?php print isset($form['group_twitter']) ? $form['group_twitter'] : ''; ?>"/>
                     </div>
                 </div>
                 <div class="create-group__input-row">
                     <div class="create-group__input-container create-group__input-container--vertical-spacing create-group__input-container--50">
-						<label class="create-group__label" for="group-telegram"><?php print __("Telegram", "community-portal"); ?></label>
+						<label class="create-group__label" for="group-telegram"><?php _e('Telegram', "community-portal"); ?></label>
```

Comment:
Community portal should be wrapped in single quotes

Reply:
Updated